src/test/run_known_gcc_test_failures.txt

```diff
@@ -34,8 +34,6 @@
 
 ### Undefined behavior: native clang also fails
 
-# Unlike gcc, libm functions are not replaced with builtins in clang.
-20030125-1.c.s.wast.wasm O2 # abort()
```

Comment:
This one is interesting.. do you know what got fixed?

Reply:
Curious too